### ISSUE TITLE
Certain parts of the PDF textbook are not expanded

### ISSUE THREAD

**Issue by Darylgolden** [nus-cs2103-AY2324S2/forum#984]
![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/9102529/6b6e872b-bf24-4dac-885a-bbd0ecfd2f4b)
![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/9102529/088fe69f-b4ec-4668-9cda-a23bbd1ad769)

For instance, on class diagrams and SOLID principles.

**Comment by damithc** [comment]
@Darylgolden collapsed topics appear again elsewhere in the PDF. They are collapsed to avoid duplication.

**Comment by YuvBindal** [comment]
Yeah, I was encountering this error just try to Command F and next page for the topic you searched you will eventually be able to find them!
